_docs/firecloud-launch-with.md

```diff
@@ -12,6 +12,8 @@ information on what that looks like from a user point of view in a mini tutorial
 When browsing WDL workflows from within Dockstore, you will see a "Launch with FireCloud" button on the right. The currently selected
 version of the workflow will be exported, but this can subsequently be changed in the FireCloud UI if need be.
 
+Note: Workflows with file-based imports are unsupported in FireCloud, the button will be disabled for those workflow versions.
```

Comment:
Similar to my previous comment; right now the button is hidden not disabled. How do we keep the code and doc in sync?

Reply:
I think we just have to describe the current situation and update when the docs change. 
Ideally, we could use comments to describe what needs to change afterwards, not sure what comment format markdown uses.

Reply:
In that case the doc for now should say `the button will be hidden for those workflow versions`.

Reply:
Could always merge after your disabled-button fix is in